### ISSUE TITLE
maven() does not work, ls() does

### ISSUE THREAD

**Issue by phdoerfler** [foundweekends/giter8#159]
# Actual

```
unfiltered_version = maven(net.databinder, unfiltered_2.11)
```

will result in `unfiltered_version` being set to `maven(net.databinder, unfiltered_2.11)`

whereas

```
specs2Version = ls(giter8-plugin, n8han)
```

will result in `specs2Version` being set to `0.6.7` correctly.
# Expected

`maven()` should correctly resolve the last version of the given artifact.

**Comment by xuwei-k** [comment]
Which giter8 version do you use? `maven()` syntax available from `0.6.7`

**Comment by eed3si9n** [comment]
I'm going to close this out.
